view_post.php

```diff
@@ -2,63 +2,57 @@
 	$my_username = "USERNAME";
 	$my_password = "PASSWORD";
 
-	mysql_connect ('localhost') ;
-	mysql_select_db ('test_db');
+	include('header.inc.php');
 
 	if (!isset($_GET['id']) || !is_numeric($_GET['id']))
 	{
 		die("Invalid ID specified.");
 	}
-	
-	$id = (int)$_GET['id'];
-	$sql = "SELECT * FROM php_blog WHERE id='$id' LIMIT 1";
 
-	$result = mysql_query($sql) or print ("Can't select entries from table php_blog.<br />" . $sql . "<br />" . mysql_error());
+	$id = (int)$_GET['id'];
+	$result = $database->query("SELECT * FROM `php_blog` WHERE `id` = '$id' LIMIT 1");
 
-	while($row = mysql_fetch_array($result))
+	foreach ($result as $row)
 	{
-		$date = date('D \t\h\e jS \of F, Y \a\t H:i',$row['timestamp']);
-		$title = stripslashes($row['title']);
-		$entry = stripslashes($row['entry']);
```

Comment:
This is why magic quotes was bad. You'd need to do this every time you output something to tidy up the data it made a mess of. Your database shouldn't have had extra slashes in it though!